ccsds/meta.json

```diff
@@ -2,21 +2,135 @@
   "name": "CCSDS",
   "submodules": [
     {
-      "name": "CCSDS Types",
+      "name": "CCSDS Packets",
       "elements": [
         {
           "name": "Packet Types",
-          "asn1Files": ["PacketTypes.asn1"]
+          "asn1Files": ["PacketTypes.asn1"],
+          "imports": [
+            {
+              "from": "ApplicationProcess",
+              "type": "ApplicationProcess-ID"
+            },
+            {
+              "from": "PacketErrorControl",
+              "type": "PlaceholderType"
+            }
+          ]
         },
         {
           "name": "TC Packet",
-          "asn1Files": ["TC-Packet.asn1"]
+          "asn1Files": ["TC-Packet.asn1"],
+          "imports": [
+            {
+              "from": "PacketTypes",
+              "type": "CCSDS-Packet"
+            },
+            {
+              "from": "PacketTypes",
+              "type": "PacketDataField"
+            },
+            {
+              "from": "PacketTypes",
+              "type": "Packet-ID"
+            },
+            {
+              "from": "PacketTypes",
+              "type": "MessageType-ID"
+            },
+            {
+              "from": "PacketTypes",
+              "type": "PacketPrimaryHeader"
+            },
+            {
+              "from": "PacketTypes",
+              "type": "UserDataField"
+            },
+            {
+              "from": "ApplicationProcessUser",
+              "type": "ApplicationProcessUser-ID"
+            },
+            {
+              "from": "TC-Payload",
+              "type": "TC-Data"
+            }
+          ]
         },
         {
           "name": "TM Packet",
-          "asn1Files": ["TM-Packet.asn1"]
+          "asn1Files": ["TM-Packet.asn1"],
+          "imports": [
+            {
+              "from": "PacketTypes",
+              "type": "CCSDS-Packet"
```

Comment:
maybe "type" should be a list? or should we remove it altogether (and keep only "from")? thoughts?

btw: it's inside same metdata, so only additional asnFile  or "requires" should be added and that's all - `imports` are from types referenced across  single  metadata boundaries

Reply:
I think it's not even needed, as we give whole asn.1 module to the compiler. Maybe we should rethink json structure?

Reply:
probably :)